linode/manager
# refactor: [M3-8183] - Clean up loading components

## Description 📝
We have a lot of loading components that are all basically just variations of CircleProgress. This PR cleans up those components.

I also moved the overage pricing tooltip text position in the Object Storage Create Bucket drawer to the top so that the tooltip doesn't block the CTA buttons

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/115299789/b4a9cfd5-b208-4992-9d06-952ccd17769e) | ![image](https://github.com/linode/manager/assets/115299789/3158b2a9-519b-4153-b570-24d3682b320d) |
| ![image](https://github.com/linode/manager/assets/115299789/c6420cea-4659-408e-94a0-b94bd048436b) | ![image](https://github.com/linode/manager/assets/115299789/40250384-9b33-4e9f-a51e-e5bd1d57b253) |
| ![image](https://github.com/linode/manager/assets/115299789/910c3f78-94e1-412a-abe4-6e6fd779b9e2) | ![image](https://github.com/linode/manager/assets/115299789/1af0714f-c225-4d4a-96c6-3a01c648f96d) |

## How to test 🧪
### Verification steps
(How to verify changes)
- Check the affected loading spinners throughout the app

## As an Author I have considered 🤔

*Check all that apply*
